docs/getting-started/index.md

```diff
@@ -7,8 +7,30 @@ slug: /getting-started
 
 # Getting Started
 
+## Overview
+
 Learn how to set up a Linux system to use zoned storage devices.
 
+## Introduction
+
+If you have never used a Shingled Magnetic Recording (SMR) disk before, this is
```

Comment:
Let's be more general here: replace "Shingled Magnetic Recording (SMR) disk" with "zoned block device". And you can make that a link to https://zonedstorage.io/docs/introduction/zoned-storage.